feeds/wlan-ap/opensync/patches/20-new-voice-video-detect-params.patch

```diff
@@ -0,0 +1,9209 @@
+--- a/src/sm/src/ucc_report.c
```

Comment:
this patch edits files inside the src folder. patch them directly, rather than adding a patch please.